plan.go

```diff
@@ -115,3 +116,42 @@ func (c *planRemove) Run(context *cmd.Context, client *cmd.Client) error {
 	fmt.Fprintf(context.Stdout, "Plan successfully removed!\n")
 	return nil
 }
+
+type planRoutersList struct{}
+
+func (c *planRoutersList) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "router-list",
+		Usage:   "router-list",
+		Desc:    "List all routers available for plan creation.",
+		MinArgs: 0,
+	}
+}
+
+func (c *planRoutersList) Run(context *cmd.Context, client *cmd.Client) error {
+	url, err := cmd.GetURL("/plan/routers")
```

Comment:
It should be ``/plans/routers``.

I was wondering whether we could add some integration tests to our clients to detect these typos in the URL or the method.